### PR TITLE
Ds workbook review

### PR DESCRIPTION
**Pull request by retabak**
- added solutions to several pages, including several data cycle pages and pages from new lessons
- fixed many typos and errors
- clarified instructions on several pages
- removed all "What's on your mind" pages
- resolved page duplication issue (pages listed in both workbook.txt and as @opt-printable) (flagged by new warnings!)
- many other little tweaks!

### CODE REVIEW

File: lessons/lookups/langs/en-us/pages/defining-rows.adoc [hunk 1]
```diff
@@ -16,46 +16,54 @@ dog-row = row-n(animals-table, 10) # Toggle is a dog
...
-@n The index of a row containing a lizard is 					@right{@fitb{5em}{}}
-@n The index of a row containing a rabbit is 					@right{@fitb{5em}{}}
-@n The index of a row containing a fixed animal is 				@right{@fitb{5em}{}}
-@n The index of a row containing a male animal 					@right{@fitb{5em}{}}
-@n The index of a row containing a female animal is 			@right{@fitb{5em}{}}
-@n The index of a row containing a hermaphroditic animal is 	@right{@fitb{5em}{}}
-@n The index of a row containing an unfixed animal is 			@right{@fitb{5em}{}}
-@n The index of a row containing a young animal (<2 years) is 	@right{@fitb{5em}{}}
-@n The index of a row containing an old animal (>10 years) is 	@right{@fitb{5em}{}}
+@n The index of a row containing a lizard is @fitb{}{@ifsoln{21 or 22}}
```
Comment (schanzer) [inline lessons/lookups/langs/en-us/pages/defining-rows.adoc:19]: just out of curiosity - why did you remove the extra spaces to make the `fitb`s line up in the source file?

Comment (retabak) [reply lessons/lookups/langs/en-us/pages/defining-rows.adoc:19]: This looked VERY WEIRD and wrong in the workbook. Like, the numbers that are supposed to go at the beginning of the line were showing up in the middle of the line, at the end of the line.... I was just trying to figure out what was going on. Ended up changing the `fitb` to extend across the whole line rather than 5em. Still not sure what the issue was?

File: lessons/measures-of-center/langs/en-us/pages/data-cycle-practice.adoc [hunk 2]
```diff
@@ -14,7 +14,7 @@ Complete both of the Data Cycles shown here, which have questions defined to get
   #:show-filter? #f
   #:build-fn ""
   #:show-build? #f
-  #:expression "mean(filter(animals-table, is-cat)), \"age\")"
+  #:expression '(mean(filter(animals-table is-cat) "age"))
```
Comment (schanzer) [inline lessons/measures-of-center/langs/en-us/pages/data-cycle-practice.adoc:17]: @retabak minor nit for the future -- we always leave space after the function name in s-expressions/wescheme. Purely stylistic, so no need to change it here)
